### PR TITLE
Fix keypair name

### PR DESCRIPTION
**Pull request by sk4zuzu**

### CODE REVIEW

File: resources/terraform/main.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
-resource "aws_key_pair" "ec2-user" {
-  key_name   = var.admin_username
+resource "aws_key_pair" "kp" {
+  key_name   = "kp-${var.name}"
```
Comment (mkyc) [inline resources/terraform/main.tf:2]: ??

```suggestion
  key_name   = "kp-${var.key_name}"
```

**Review by mkyc — CHANGES_REQUESTED** [review]
I didn't tested it but looks like there is small bug.

File: resources/terraform/main.tf (hunk 1 above)
Comment (mkyc) [reply resources/terraform/main.tf:2]: Or maybe variable `key_name` should be removed?

Comment (sk4zuzu) [reply resources/terraform/main.tf:2]: `key_name` exists only inside the module. It's just a name of the resource in AWS.

Comment (mkyc) [reply resources/terraform/main.tf:2]: Ok, I checked it and it's ok.

**Review by mkyc — APPROVED** [review]
LGTM

**Review by rafzei — APPROVED** [review]

**Pull request merged**